src/api/VoteMonitor.Api.Form/Models/FormVersionsModel.cs

```diff
@@ -25,5 +25,8 @@ public class FormDetailsModel
 
         [JsonProperty(PropertyName = "diaspora")]
         public bool Diaspora { get; set; }
+
+        [JsonProperty(PropertyName = "order")]
```

Comment:
this is not useful at all
json serializer settings are lowercasing by default..

Reply:
and it's available for all the other attributes as well, I know this one was added just to have consistency